### PR TITLE
feat(planner) Implement UI for home pages

### LINKED ISSUE #135: [Feature Request] Add view for all degree plans
# Overview

After logging into Planner, the user should be able to see all the plans that they have created. This component has already been implemented in the codebase. It just needs to be slightly tweaked to match the current design.

# Proposal

Implement the following view & integrate into the home page.

Degree Plan Page Drafttests

### LINKED ISSUE #138: [Feature Request] Add "Credits" view for Home page
# Overview

After logging into Planner, the user should be able to see all the course credits that they have taken & entered into Planner.

# Proposal

Implement the following view & integrate into the home page.

Credits Pagesave

### PR DESCRIPTION
**Pull request by ZzRanger**
## Overview

Implement UI for the Home Pages. This includes "Home", "Degrees", "Profile", and "Credits".
Resolves #135 and #138 .

Functionality for all these screens still need to be completed. Additionally, mockups for the "Home" screen warrant a further discussion and may be need to be updated as well.

## What Changed

Below is what the home pages now look like.
Screen Shot 2022-09-04 at 12 43 25 AM
Screen Shot 2022-09-04 at 12 43 32 AM
Screen Shot 2022-09-04 at 12 43 40 AM
Screen Shot 2022-09-04 at 12 43 47 AM

### CODE REVIEW

File: components/home/plans/PlanCard.tsx [hunk 1]
```diff
@@ -16,11 +18,28 @@ export default function PlanCard({ id, plan }: PlanCardProps) {
...
-      <div className="text-subtitle2">Major: {major}</div>
-      <div className="text-sm">{semesters.length} semesters</div>
+      <div className="relative">
+        <div className="text-headline6">{title}</div>
+        <div className="absolute -right-10 -top-1 text-white">
+          <IconButton
+            color="inherit"
+            onClick={(e) => {
+              e.stopPropagation();
+              alert('HI');
```
Comment (rajmeet2001) [inline components/home/plans/PlanCard.tsx:30]: What is this for?

File: pages/app/profile.tsx [hunk 2]
```diff
@@ -14,23 +18,139 @@ export default function ProfilePage(): JSX.Element {
...
+            <div className="self-end">heh</div>
+          </div>
+          <div className="flex flex-col">
+            <div className="flex flex-row justify-between">
+              <div className="text-2xl">Minor</div>
+              <IconButton
+                color="inherit"
+                onClick={(e) => {
+                  e.stopPropagation();
+                  alert('HI');
```
Comment (rajmeet2001) [inline pages/app/profile.tsx:80]: It comes again

**Comment by ZzRanger** [comment]
Put that in when I was testing out the UI component. Removed the alerts with the latest commit.

**Review by rajmeet2001 — DISMISSED** [review]
Last approval as Lead :(

**Pull request merged**